tests/app/test_retriever.py

```diff
@@ -36,11 +36,11 @@ def add_secure_message(self, msg_id, subject="test", body="test", thread_id="Thr
                     '{thread_id}', '{collection_case}', '{ru_id}', '{survey}', '{collection_exercise}', '{from_internal}')'''
             con.execute(query)
 
-    def add_conversation(self, conversation_id):
+    def add_conversation(self, conversation_id, is_closed):
```

Comment:
The api doesn't allow creating a conversation in a closed state, so I don't think it should be a parameter here i.e. the SQL should always set `is_closed` to `False`.